Fabric.Identity.API/Validation/ApiResourceValidator.cs

```diff
@@ -22,9 +22,14 @@ private void ConfigureRules()
 
             RuleFor(apiResource => apiResource.Scopes)
                 .NotNull()
-                .SetCollectionValidator(new ScopeValidator())
                 .WithMessage("Please specify at least one Scope for this Api Resource");
 
+            RuleForEach(apiResource => apiResource.Scopes)
+                .SetValidator(new ScopeValidator())
+                .WithMessage("Please specify at least one Scope for this Api Resource");
```

Comment:
Is this rule being duplicated?

Reply:
No, it's not. One of the validators checks if the collection is null, and this one checks if there are items in the collection and that those items are valid.